specification/introduction/introduction.tex

```diff
@@ -1,7 +1,11 @@
 \chapter{Introduction}\label{sec:introduction}
 
-This chapter covers the basic concepts that govern development and maintenance of the specification.
-The actual specification is contained in the following chapters.
+UAVCAN is a lightweight protocol designed to provide a highly reliable communication method for
+aerospace and robotic applications via robust vehicle bus networks.
+
+\begin{mdframed}
+    This is a non-nomative section covering the basic concepts that govern development and maintenance of the specification.
+\end{mdframed}
```

Comment:
Can we please get rid of the frame? ;) Also the `r` I mentioned earlier still seems to be missing.